<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QmitkMLBTViewControls</class>
 <widget class="QWidget" name="QmitkMLBTViewControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>1127</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolBox" name="toolBox">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>359</width>
        <height>1065</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Training</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0">
        <widget class="QCommandLinkButton" name="m_LoadForestButton">
         <property name="text">
          <string>Load Forest</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="4" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Step size:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Num. Trees:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Max. Depth:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QDoubleSpinBox" name="m_TrainingStepSizeBox">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="minimum">
             <double>-1.000000000000000</double>
            </property>
            <property name="maximum">
             <double>999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>-1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="m_NumTreesBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Sample Fraction:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="m_MaxDepthBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="m_GmSamplingBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>999999999</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="m_SampleFractionBox">
            <property name="decimals">
             <number>3</number>
            </property>
            <property name="maximum">
             <double>1.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>GM Sampling Points:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Use Previous Direction:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="m_TrainingUsePreviousDirectionBox">
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QCommandLinkButton" name="m_StartTrainingButton">
         <property name="text">
          <string>Start Training</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QCommandLinkButton" name="m_SaveForestButton">
         <property name="text">
          <string>Save Forest</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>359</width>
        <height>1065</height>
       </rect>
      </property>
      <attribute name="label">
       <string>Tractography</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="0">
        <widget class="QCheckBox" name="m_TrackingUsePreviousDirectionBox">
         <property name="text">
          <string>Use Previous Direction</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QFrame" name="frame_5">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="1">
           <widget class="QToolButton" name="m_PauseTrackingButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../../../../debug-binary/MBI-build/Plugins/org.mbi.gui.qt.diffusionimaginginternal/org_mbi_gui_qt_diffusionimaginginternal_cached.qrc">
              <normaloff>:/org.mbi.gui.qt.diffusionimaginginternal/resources/Media-playback-pause.svg</normaloff>:/org.mbi.gui.qt.diffusionimaginginternal/resources/Media-playback-pause.svg</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QToolButton" name="m_StartTrackingButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../../../../debug-binary/MBI-build/Plugins/org.mbi.gui.qt.diffusionimaginginternal/org_mbi_gui_qt_diffusionimaginginternal_cached.qrc">
              <normaloff>:/org.mbi.gui.qt.diffusionimaginginternal/resources/Media-playback-start.svg</normaloff>:/org.mbi.gui.qt.diffusionimaginginternal/resources/Media-playback-start.svg</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QToolButton" name="m_AbortTrackingButton">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../../../../debug-binary/MBI-build/Plugins/org.mbi.gui.qt.diffusionimaginginternal/org_mbi_gui_qt_diffusionimaginginternal_cached.qrc">
              <normaloff>:/org.mbi.gui.qt.diffusionimaginginternal/resources/Media-playback-stop.svg</normaloff>:/org.mbi.gui.qt.diffusionimaginginternal/resources/Media-playback-stop.svg</iconset>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QFrame" name="frame_4">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <widget class="QCheckBox" name="m_DemoModeBox">
            <property name="text">
             <string>Demo Mode</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="m_TimerIntervalBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>10</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="m_TrackingStepSizeBox">
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Max. Length</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="m_NumSamplesBox">
            <property name="maximum">
             <number>999999999</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Num. Samples:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Input DWI:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Step Size:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Sampling Distance:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="m_NumberOfSeedsBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Min. Length</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDoubleSpinBox" name="m_SamplingDistanceBox">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QDoubleSpinBox" name="m_MinLengthBox">
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
            <property name="value">
             <double>20.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Angular Threshold:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Num. Seeds:</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QDoubleSpinBox" name="m_MaxLengthBox">
            <property name="maximum">
             <double>999999999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
            <property name="value">
             <double>400.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QDoubleSpinBox" name="m_AngularThresholdBox">
            <property name="decimals">
             <number>1</number>
            </property>
            <property name="maximum">
             <double>90.000000000000000</double>
            </property>
            <property name="value">
             <double>45.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QmitkDataStorageComboBox" name="m_TrackingRawImageBox"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Num. Threads:</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QSpinBox" name="m_NumberOfThreadsBox">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>30</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="10" column="0">
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="3" column="0">
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_5">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item row="6" column="0">
           <widget class="QCheckBox" name="m_TrackingUseStopImageBox">
            <property name="text">
             <string>Use Stop Image:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="m_TrackingUseSeedImageBox">
            <property name="text">
             <string>Use Seed Image:</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="m_TrackingUseMaskImageBox">
            <property name="text">
             <string>Use Mask Image:</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QmitkDataStorageComboBox" name="m_TrackingMaskImageBox"/>
          </item>
          <item row="4" column="1">
           <widget class="QmitkDataStorageComboBox" name="m_TrackingSeedImageBox"/>
          </item>
          <item row="6" column="1">
           <widget class="QmitkDataStorageComboBox" name="m_TrackingStopImageBox"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QmitkDataStorageComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">QmitkDataStorageComboBox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../../../debug-binary/MBI-build/Plugins/org.mbi.gui.qt.diffusionimaginginternal/org_mbi_gui_qt_diffusionimaginginternal_cached.qrc"/>
 </resources>
 <connections/>
</ui>
